test/crawler_web/features/authentication/register_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule CrawlerWeb.Authentication.RegisterTest do
+  use CrawlerWeb.FeatureCase, async: true
+
+  alias CrawlerWeb.Endpoint
+
+  feature "when registering a new account successfully, renders home page", %{session: session} do
+    user = build(:user)
+    authenticated_session = register_user(session, user)
+
+    assert current_path(authenticated_session) == Routes.page_path(Endpoint, :index)
+
+    authenticated_session
+    |> assert_has(Query.text(user.email))
+    |> assert_has(Query.text("Log out"))
+  end
+
+  feature "when failed to register new account, renders register page with error", %{
```

Comment:
Just FYI normally we don't test the unhappy path on UI test as it's slower than Unitest :)

Reply:
Removed in [5939988](https://github.com/vnntsu/elixir-ic/pull/32/commits/5939988e956f649bc22c5b242696a2a6408c284c)